src/utils.ts

```diff
@@ -238,22 +129,10 @@ export type StrictEqualUsingTSInternalIdenticalToOperator<L, R> =
     : false
 
 /**
- * Checks if two types are strictly equal using branding.
- */
-export type StrictEqualUsingBranding<Left, Right> = And<
-  [ExtendsUsingBranding<Left, Right>, ExtendsUsingBranding<Right, Left>]
->
-
-/**
- * Represents a type that checks if two types are equal, using
- * a hopefully performant approach.
- * It first checks if the types are strictly equal using
- * {@linkcode StrictEqualUsingTSInternalIdenticalToOperator}.
- * If they are not strictly equal, it falls back to using the
- * {@linkcode StrictEqualUsingBranding} type.
+ * Checks that `L` and `R` extend each other. Not quite the same as an equality check since `any` can make it resolve to true.
+ * So should only be used when `L` and `R` are known to avoid `any`.
  */
-export type HopefullyPerformantEqual<Left, Right> =
```

Comment:
Removed because it's unused. This was a type that I think I originally intended to avoid the need to add `.branded` manually for types that were tripped up by intersections, but I think I remember it not working reliably so am removing. Highly unlikely it's used by anyone.